h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -3306,6 +3306,54 @@ def check_sorted_2_columns(frame1, sorted_column_indices, prob=0.5, ascending=[T
                                                                                              "row {2}: {3}".format(rowInd, frame1[rowInd,colInd],
                                                                                                                    rowInd+1, frame1[rowInd+1,colInd])
 
+def check_sorted_2_columns(frame1, sorted_column_indices, prob=0.5, ascending=[True, True]):
+    for colInd in sorted_column_indices:
+        for rowInd in range(0, frame1.nrow-1):
+            if (random.uniform(0.0,1.0) < prob):
```

Comment:
I really really dont like this patter, if something is wrong this test will fail only half of the time

Reply:
Honza:  You can set prob=1.0 and this test will check row.  I am doing this to save time.  Copied this from Arno originally.

Reply:
I know its copied everywhere, but that makes me like it even less